src/services/user.validation.ts

```diff
@@ -11,27 +11,28 @@ export const createUserSchema = Joi.object({
     age: Joi.number().required().min(4).max(130)
 })
 
-export interface ICreateUserSchema extends joiValidation.ValidatedRequestSchema {
-    [joiValidation.ContainerTypes.Body]: IBaseUser
-}
-
 export const editUserSchema = Joi.object({
     login: Joi.string(),
     password: Joi.string().regex(/[0-9][a-z]/),
     age: Joi.number().min(4).max(130),
-    id: Joi.string().required()
+    id: Joi.number().required()
```

Comment:
Some expressions can be saved as variables and reused. For example `Joi.number().min(4).max(130),`

Reply:
how we can know how much ids we have? i'd say that it will be reduntable checking